{"classFile":"FailOnTimeoutTest","className":"FailOnTimeoutTest","packages":"org.junit.internal.runners.statements","methodInfos":[{"md5":"Mzfy6cU8f6XQqNYy4Lu1MA==","methodName":"throwExceptionIfTheSecondCallToEvaluateNeedsTooMuchTime","parameters":"[]"},{"md5":"KFoP8jaecSjQkW1TgEIDhw==","methodName":"throwTimeoutExceptionOnSecondCallAlthoughFirstCallThrowsException","parameters":"[]"}],"addLines":[84,98,252],"delLines":[84,98,252],"type":"MODIFY","path":"src/test/java/org/junit/internal/runners/statements/FailOnTimeoutTest.java","pro_Commit":"de77f666b6bcc640425dbbca559c9d62ea64a815","pro_Date":"2021-01-10 12:14:02","sha1":"04b329b1fff2f3c1b12e0ea5aa1eb721b8880b2f","file_Date":"2021-01-20 14:12:53","projectName":"junit4","commitID":"d27ad5259228e84c235dac24cd62f974ac0a8e1d"}